compiler/src/dotty/tools/dotc/core/tasty/TastyFormat.scala

```diff
@@ -252,7 +253,7 @@ Standard Section: "Comments" Comment*
 object TastyFormat {
 
   final val header: Array[Int] = Array(0x5C, 0xA1, 0xAB, 0x1F)
-  val MajorVersion: Int = 17
+  val MajorVersion: Int = 18
```

Comment:
The `open` modifier changes are currently in master without receiving a tasty version bump,  so possibly there should be a prior bump before this one, a possible consideration.

Reply:
I think we can bump once for both.

Reply:
sounds good